web/src/engine/keyboard/src/keyboards/loaders/domKeyboardLoader.ts

```diff
@@ -84,4 +75,14 @@ export class DOMKeyboardLoader extends KeyboardLoaderBase {
     // being ignored.
     return uri + "?v=" + (new Date()).getTime(); /*cache buster*/
   }
+
+  private evalScriptInContext(script: string, context: any) {
+    const f = function (s: string) {
+      // use indirect eval (eval?.() notation doesn't work because of esbuild bundling)
+      const evalFunc = eval;
+      return evalFunc(s);
+    }
+    f.call(context, script);
+  }
```

Comment:
It would be good to have a test to verify that this actually does what we expect with the `context` parameter.